src/features/unstake/Unstake.tsx

```diff
@@ -27,6 +28,12 @@ export const Unstake = () => {
         fromToken="stCELO"
         toToken="CELO"
       />
+      {pendingWithdrawals.length > 0 ? (
```

Comment:
Could it be && operator? Cant remember if react renders boolean value.

Reply:
Treats `false` the same as `null` :+1: